### DISCUSSION TITLE
Welcome to being-an-WoB'24 Discussions!

### DISCUSSION THREAD

**Discussion by Durgesh4993** [Girlscript-WOB/Contributor#1] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

Ask questions you’re wondering about.

- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded. Remember that this is a community we
- build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by techy4shri** [comment]
Hello, Garima here. I know git and code and just swim here in all kinds of tech honestly. 
I was part of GSSoC 2024 before so I know how things happen in a way. 
Don't be shy and ask for help if you need! I will be happy to lend a hand!
